cogs/vxtwitconverter/vxtwitconverter.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import os
+
+from discord import Message, channel
+from urlextract import URLExtract
+
+from redbot.core import commands, data_manager
+from redbot.core.bot import Red
+
+
+class VxTwitConverter(commands.Cog):
+    """Converts twitter link embeds to vxtwitter"""
```

Comment:
Similar to my previous comment. Quickly describe the effect of VxTwitter here. Something like

```suggestion
    """Converts Twitter link to VxTwitter for better video embeds."""
```

should be enough.